DOCS add note to changelog for Solr changes (#9576)
silverstripe/silverstripe-framework: docs/en/04_Changelogs/4.6.0.md

 * [MySQL tables are auto-converted from MyISAM to InnoDB](#myisam)
 * [Editing files directly in the insert-media modal](#in-modal-editing)
 * [MIME Type validation now a core module](#mime-validator)

>  * [Solr no longer indexes draft/ restricted content](#solr-updates)

Reviewer: ```suggestion
 * [Solr no longer indexes draft/restricted content](#solr-updates)
```

Author: Updated

Read [Allowed file types](Developer_Guides/Files/Allowed_file_types) in the
Silverstripe CMS doc for all the details.

> ## Solr no longer indexes draft/ restricted content {#solr-updates}

Reviewer: ```suggestion
## Solr no longer indexes draft/restricted content {#solr-updates}
```

Author: Updated

> At the time of this release a new version of the popular [FullTextSearch module](https://github.com/silverstripe/silverstripe-fulltextsearch) is also available, introducing more secure defaults. Most notably, draft and restricted content will no longer be indexed by default, due to a `canView()` check being performed against an anonymous user prior to (re)indexing. Restricted content means that it has a permission level of either 'Logged-in users' or 'Only these groups'.

Reviewer: ```suggestion
At the time of this release a new version of the popular [silverstripe/fulltextsearch module](https://github.com/silverstripe/silverstripe-fulltextsearch) is also available, introducing more secure defaults. Most notably, draft and restricted content will no longer be indexed by default, due to a `canView()` check being performed against an anonymous user prior to (re)indexing. Restricted content means that it has a permission level of either 'Logged-in users' or 'Only these groups'.
```

Author: Updated

If your project uses this module, after upgrading your website, ensure that you run the `Solr_Reindex` task on your production environment to remove previously indexed content that should no longer be there.

If your website requires draft or restricted content to be indexed, you can opt-out of the new secure defaults on a per-model basis.

> This is a great opportunity to make sure that any custom indexes / search controllers in your project are correctly filtering results based on permissions and search visibility, which you can now achieve via a unified method (see `SilverStripe\FullTextSearch\Search\ServicesSearchableService::isSearchable()`.)

Reviewer: ```suggestion
This is a great opportunity to make sure that any custom indexes/search controllers in your project are correctly filtering results based on permissions and search visibility, which you can now achieve via a unified method (see `SilverStripe\FullTextSearch\Search\ServicesSearchableService::isSearchable()`.)
```

Author: Updated

> This is a great opportunity to make sure that any custom indexes/search controllers in your project are correctly filtering results based on permissions and search visibility, which you can now achieve via a unified method (see `SilverStripe\FullTextSearch\Search\ServicesSearchableService::isSearchable()`.)

Reviewer 2: ```suggestion
This is a great opportunity to make sure that any custom indexes/search controllers in your project are correctly filtering results based on permissions and search visibility, which you can now achieve via a unified method (see `SilverStripe\FullTextSearch\Search\Services\SearchableService::isSearchable()`.)
```

> The [FullTextSearch module Readme provides additional information](https://github.com/silverstripe/silverstripe-fulltextsearch).

Reviewer: ```suggestion
The [silverstripe/fulltextsearch module readme provides additional information](https://github.com/silverstripe/silverstripe-fulltextsearch).
```

Author: Updated
